.dockerignore

```diff
@@ -1,13 +1,20 @@
-**/node_modules/
-**/node_modules_linux/
+**/.circleci/
+**/.vscode/
+**/.idea/
```

Comment:
Why include this?  We don't have this.

Reply:
My IDE is Webstorm (Jetbrain Co.) and it produces this file like .vscode for Visual Studio, I thought maybe some other people are using this. I have the same problem with Git. Everybody always includes .vscode only :)
I will remove this if it's not popular.

Reply:
That's fine.  Should we be adding this to our `.gitignore` too then?

Reply:
That would make my life easier. Currently, I can't do `git add .` 😀

Reply:
Always feel free to add things to `.gitignore` that would help you with your particular environment.  We have lots of stuff in there that's only specific to some people in the project, but not all.  We want everyone to have their life be easier :)